content/posts/gitops/index.md

```diff
@@ -0,0 +1,497 @@
+---
```

Comment:
Most of the images would make more sense as PNGs instead of SVGs (e.g. the ones that contain screenshots).

Reply:
Agreed. Transformed the screenshots to PNGs.